examples/resources/couchbase-capella_cluster_onoff_schedule/resource.tf

```
@@ -0,0 +1,14 @@
+resource "couchbase-capella_cluster_onoff_schedule" "new_cluster_onoff_schedule" {
+  organization_id = "organization_id"
+  project_id      = "project_id"
+  cluster_id      = "cluster_id"
+  timezone        = "timezone(ex:US/Hawaii)"
+  days = [
+    for day in var.days : {
+      state = "state"
```

@SaicharanCB can you add some of the original values? Check from the README. The users will get confused otherwise. Note: that the ids are generic for security.
```suggestion
      state = "off"
```